test/util/spec.helpers.js

```diff
@@ -0,0 +1,117 @@
+var helpers = require('../../lib/util/helpers');
+
+describe('helpers', function () {
+
+    describe('getPreviousSteps', function () {
+        var getPreviousSteps, steps;
+
+        beforeEach(function () {
+            getPreviousSteps = helpers.getPreviousSteps;
+        });
+
+        it('is a function', function () {
+            getPreviousSteps.should.be.a('function');
+        });
+
+        it('returns an array', function () {
+            getPreviousSteps().should.be.an('array');
+        });
+
```

Comment:
How about the above two tests and then two more:

`it('returns an empty array when there are no matching previous steps')`
`it('returns an array of all routes from `next` and `forks` properties')` - you could have three parts within this that test for just next, just forks and then both. You should be able to build one steps object than can satisfy all these cases.